app/static/js/layers.js

```diff
@@ -431,6 +432,6 @@ let ogclustersTileLayer = L.vectorGrid.protobuf("https://tile.rl-institut.de/dat
 ogclustersTileLayer.on("click", function (event) {
   map.options.maxZoom = 19;
   village_button_fun();
-  map.flyTo([event.latlng.lat, event.latlng.lng], 14);
```

Comment:
You don't include the 14 in the change? was it the zoom level?

Reply:
Yes it was. It's now chosen automatically by the bounds